Nodejs/Product/Nodejs/SharedProject/CommonProjectNode.cs

```diff
@@ -431,59 +412,18 @@ private void BoldStartupItem()
             }
         }
 
-        private FileSystemWatcher CreateFileSystemWatcher(string dir)
+        private void CreateFileSystemWatcher()
```

Comment:
>CreateFileSystemWatcher [](start = 21, length = 23)

Rename?

Reply:
Also, what does this do?

---
In reply to: [160539363](https://github.com/Microsoft/nodejstools/pull/1835#discussion_r160539363) [](ancestors = 160539363)